Generation/src/components/MomentumRangeParticleGun.cpp

```diff
@@ -9,12 +9,7 @@
 #include "GaudiKernel/PhysicalConstants.h"
 #include "GaudiKernel/IRndmGenSvc.h" 
 
-//===========================================================================
-// Implementation file for class: MomentumRangeParticleGun
-//
-// 2008-05-18: Patrick Robbe adaptation to tool structure
-// 2014-06-22: Benedikt Hegner adaption for non LHCb use cases
-//===========================================================================
```

Comment:
Oh, I remember now why I removed it. It was already as a doxygen comment in the header (where it belongs, I guess) I updated it with the changes.